app/Http/PsrRouterServiceProvider.php

```diff
@@ -596,4 +600,11 @@ private function bindMailinglist(Router $router): void
     {
         $router->put('mailing-list/{mailingListId}/', SubscribeUserToMailinglistRequestHandler::class);
     }
+
+    private function bindTaxonomyEndpoints(Router $router): void
```

Comment:
Also, here we may need to add the `cultuurkuur` group to provide `/cultuurkuur/regions/` and that would change the name of `bindTaxonomyEndpoints` to `bindCultuurkuur`